integration-tests/integration/coordinator/remove-involved-staff.spec.js

```diff
@@ -44,12 +45,12 @@ context('A use of force coordinator can remove involved staff', () => {
 
     seedReport()
 
-    let allIncidentsPage = AllIncidentsPage.goTo()
-    allIncidentsPage.getTodoRows().should('have.length', 1)
-    allIncidentsPage.getNoCompleteRows().should('exist')
+    let notCompletedIncidentsPage = NotCompletedIncidentsPage.goTo()
+    notCompletedIncidentsPage.getTodoRows().should('have.length', 1)
+    notCompletedIncidentsPage.getNoCompleteRows().should('not.exist')
```

Comment:
`getNoCompleteRows ` would never exist as "no completed rows" will never show on the incomplete page.

This function will need to be removed from the `notCompletedIncidentsPage` and can check using a new mechanism using the DB for accessing report count

Reply:
check for 'no completed rows' removed from spec.
getNoCompleteRows function removed from notCompletedIncidentsPage